coalib/bearlib/spacing/SpacingHelper.py

```diff
@@ -86,11 +86,28 @@ def replace_spaces_with_tabs(self, line: str):
         :param line: The string with spaces to replace.
         :return:     The converted string.
         """
+        def previous_whitespace():
+            # Find the previous real character, and remaining chars to fill tab
```

Comment:
oh shit sry for missing your comments...
and yes module level is okay, though you can also use `@staticmethod` instead of `@classmethod` for class-level :+1: